hera_cal/io.py

```diff
@@ -678,6 +658,61 @@ def iterate_over_times(self, Nints=1, times=None):
             yield self.read(times=times[i:i + Nints])
 
 
+def read_filter_cache(cache_dir):
+    """
+    Load files from a cache specified by cache_dir.
+
+    Parameters
+    ----------
+    cache_dir, string, path to a folder that is used for the cache
+        files in this folder with an extension .filter_cache are assumed
+        to be cache files. These files are pickled caches from previous filtering runs.
+    """
+    # Load up the cache file with the most keys (precomputed filter matrices).
+    cache = {}
+    cache_files = glob.glob(cache_dir + '/*.filter_cache')
+    # loop through cache files, load them.
+    # If there are new keys, add them to internal cache.
+    # If not, delete the reference matrices from memory.
+    for cache_file in cache_files:
+        cfile = open(cache_file, 'rb')
+        cache_t = pickle.load(cfile)
+        for key in cache_t:
+            if key not in cache:
+                cache[key] = cache_t[key]
+    return cache
+
+
+def write_filter_cache(filter_cache, cache_dir=None, skip_keys=None):
+    """
+    write cached cache to a new cache file.
```

Comment:
more documentation about what chaches are and what they're used for would be helpful

Reply:
I've added a description for both descriptions.